src/js/lotto.js

```diff
@@ -0,0 +1,174 @@
+import { isNumber } from "./util";
+
+export default class Lotto {
+  // 로또 번호의 갯수
+  #LOTTO_NUMBER_COUNT = 6;
+  #RANK = {
+    1: {
+      rank: 1,
+      reward: 2000000000,
```

Comment:
큰 숫자(긴 숫자)는 numeric seprator를 사용해보시면 좋을거 같아요!
```suggestion
      reward: 2_000_000_000,
```